app/src/main/java/app/revanced/integrations/shared/GmsCoreSupport.java

```diff
@@ -71,12 +71,14 @@ private static void showDoNotKillMyAppDialog(Context context, String messageKey)
         // Otherwise if device is in dark mode the dialog is missing a dark mode color scheme.
         Utils.runOnMainThreadDelayed(() -> {
             new AlertDialog.Builder(context)
+                    .setIcon(android.R.drawable.ic_dialog_alert)
                     .setTitle(str("gms_core_not_whitelisted_title"))
                     .setMessage(str(messageKey))
                     .setPositiveButton(android.R.string.ok, (dialog, id) -> {
                         open(DONT_KILL_MY_APP_LINK);
                         System.exit(0);
                     })
+                    .setCancelable(false)
```

Comment:
If we intend the user to dismiss it, we should add a close/continue anyways/later button. The user might be in a hurry for example where this would be better than asking the user to change some settings.

Reply:
A proper affordance instead of a hidden, seemingly unintended bypass is acceptable because the next time they launch the app, the dialogue will nag them again. The only issue I can think of is if the user skips it and experiences problems while using the app, then proceeds to report an issue. Users allegedly experience playback issues while GmsCore is killed in the background, although I hardly believe that as that would crash the app.